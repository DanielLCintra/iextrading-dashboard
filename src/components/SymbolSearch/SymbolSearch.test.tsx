// import { mount } from 'enzyme';
// import * as React from 'react';
// import SymbolSearch from './SymbolSearch';

describe('SymbolSearch', () => {
  // const wrapper = mount(<SymbolSearch />);

  it('renders without crashing', () => {
    // expect(wrapper).toMatchSnapshot();
  });
});
