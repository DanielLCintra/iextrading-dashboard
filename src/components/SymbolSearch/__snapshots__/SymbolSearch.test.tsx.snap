// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SymbolSearch renders without crashing 1`] = `
<SymbolSearch>
  <div>
    <Autosuggest
      alwaysRenderSuggestions={false}
      focusInputOnSuggestionClick={true}
      getSuggestionValue={[Function]}
      highlightFirstSuggestion={false}
      id="1"
      inputProps={
        Object {
          "onChange": [Function],
          "placeholder": "Search",
          "value": "",
        }
      }
      multiSection={false}
      onSuggestionsClearRequested={[Function]}
      onSuggestionsFetchRequested={[Function]}
      renderInputComponent={[Function]}
      renderSuggestion={[Function]}
      renderSuggestionsContainer={[Function]}
      shouldRenderSuggestions={[Function]}
      suggestions={Array []}
      theme={
        Object {
          "container": undefined,
          "suggestion": undefined,
          "suggestionsContainerOpen": undefined,
          "suggestionsList": undefined,
        }
      }
    >
      <Autowhatever
        getSectionItems={[Function]}
        highlightedItemIndex={null}
        highlightedSectionIndex={null}
        id="1"
        inputProps={
          Object {
            "onBlur": [Function],
            "onChange": [Function],
            "onFocus": [Function],
            "onKeyDown": [Function],
            "placeholder": "Search",
            "value": "",
          }
        }
        itemProps={[Function]}
        items={Array []}
        multiSection={false}
        renderInputComponent={[Function]}
        renderItem={[Function]}
        renderItemData={
          Object {
            "query": "",
          }
        }
        renderItemsContainer={[Function]}
        renderSectionTitle={[Function]}
        theme={
          Object {
            "container": undefined,
            "item": undefined,
            "itemsContainerOpen": undefined,
            "itemsList": undefined,
          }
        }
      >
        <div
          aria-expanded={false}
          aria-haspopup="listbox"
          aria-owns="react-autowhatever-1"
          key="react-autowhatever-1-container"
          role="combobox"
          style={Object {}}
        >
          <TextField
            InputProps={
              Object {
                "classes": Object {
                  "root": undefined,
                },
                "inputRef": [Function],
              }
            }
            aria-activedescendant={null}
            aria-autocomplete="list"
            aria-controls="react-autowhatever-1"
            autoComplete="off"
            fullWidth={true}
            key="react-autowhatever-1-input"
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            placeholder="Search"
            required={false}
            select={false}
            style={Object {}}
            type="search"
            value=""
          >
            <WithStyles(FormControl)
              aria-activedescendant={null}
              aria-autocomplete="list"
              aria-controls="react-autowhatever-1"
              fullWidth={true}
              onKeyDown={[Function]}
              required={false}
              style={Object {}}
            >
              <FormControl
                aria-activedescendant={null}
                aria-autocomplete="list"
                aria-controls="react-autowhatever-1"
                classes={
                  Object {
                    "fullWidth": "MuiFormControl-fullWidth-4",
                    "marginDense": "MuiFormControl-marginDense-3",
                    "marginNormal": "MuiFormControl-marginNormal-2",
                    "root": "MuiFormControl-root-1",
                  }
                }
                component="div"
                disabled={false}
                error={false}
                fullWidth={true}
                margin="none"
                onKeyDown={[Function]}
                required={false}
                style={Object {}}
              >
                <div
                  aria-activedescendant={null}
                  aria-autocomplete="list"
                  aria-controls="react-autowhatever-1"
                  className="MuiFormControl-root-1 MuiFormControl-fullWidth-4"
                  onKeyDown={[Function]}
                  style={Object {}}
                >
                  <WithStyles(Input)
                    autoComplete="off"
                    classes={
                      Object {
                        "root": undefined,
                      }
                    }
                    fullWidth={true}
                    inputRef={[Function]}
                    onBlur={[Function]}
                    onChange={[Function]}
                    onFocus={[Function]}
                    placeholder="Search"
                    type="search"
                    value=""
                  >
                    <Input
                      autoComplete="off"
                      classes={
                        Object {
                          "disabled": "MuiInput-disabled-8",
                          "error": "MuiInput-error-10",
                          "focused": "MuiInput-focused-7",
                          "formControl": "MuiInput-formControl-6",
                          "fullWidth": "MuiInput-fullWidth-12",
                          "input": "MuiInput-input-13",
                          "inputMarginDense": "MuiInput-inputMarginDense-14",
                          "inputMultiline": "MuiInput-inputMultiline-15",
                          "inputType": "MuiInput-inputType-16",
                          "inputTypeSearch": "MuiInput-inputTypeSearch-17",
                          "multiline": "MuiInput-multiline-11",
                          "root": "MuiInput-root-5",
                          "underline": "MuiInput-underline-9",
                        }
                      }
                      disableUnderline={false}
                      fullWidth={true}
                      inputRef={[Function]}
                      multiline={false}
                      onBlur={[Function]}
                      onChange={[Function]}
                      onFocus={[Function]}
                      placeholder="Search"
                      type="search"
                      value=""
                    >
                      <div
                        className="MuiInput-root-5 MuiInput-fullWidth-12 MuiInput-formControl-6 MuiInput-underline-9"
                      >
                        <input
                          aria-invalid={false}
                          autoComplete="off"
                          className="MuiInput-input-13 MuiInput-inputType-16 MuiInput-inputTypeSearch-17"
                          disabled={false}
                          onBlur={[Function]}
                          onChange={[Function]}
                          onFocus={[Function]}
                          placeholder="Search"
                          required={false}
                          type="search"
                          value=""
                        />
                      </div>
                    </Input>
                  </WithStyles(Input)>
                </div>
              </FormControl>
            </WithStyles(FormControl)>
          </TextField>
          <WithStyles(Paper)
            id="react-autowhatever-1"
            key="react-autowhatever-1-items-container"
            role="listbox"
            square={true}
            style={Object {}}
          >
            <Paper
              classes={
                Object {
                  "elevation0": "MuiPaper-elevation0-20",
                  "elevation1": "MuiPaper-elevation1-21",
                  "elevation10": "MuiPaper-elevation10-30",
                  "elevation11": "MuiPaper-elevation11-31",
                  "elevation12": "MuiPaper-elevation12-32",
                  "elevation13": "MuiPaper-elevation13-33",
                  "elevation14": "MuiPaper-elevation14-34",
                  "elevation15": "MuiPaper-elevation15-35",
                  "elevation16": "MuiPaper-elevation16-36",
                  "elevation17": "MuiPaper-elevation17-37",
                  "elevation18": "MuiPaper-elevation18-38",
                  "elevation19": "MuiPaper-elevation19-39",
                  "elevation2": "MuiPaper-elevation2-22",
                  "elevation20": "MuiPaper-elevation20-40",
                  "elevation21": "MuiPaper-elevation21-41",
                  "elevation22": "MuiPaper-elevation22-42",
                  "elevation23": "MuiPaper-elevation23-43",
                  "elevation24": "MuiPaper-elevation24-44",
                  "elevation3": "MuiPaper-elevation3-23",
                  "elevation4": "MuiPaper-elevation4-24",
                  "elevation5": "MuiPaper-elevation5-25",
                  "elevation6": "MuiPaper-elevation6-26",
                  "elevation7": "MuiPaper-elevation7-27",
                  "elevation8": "MuiPaper-elevation8-28",
                  "elevation9": "MuiPaper-elevation9-29",
                  "root": "MuiPaper-root-18",
                  "rounded": "MuiPaper-rounded-19",
                }
              }
              component="div"
              elevation={2}
              id="react-autowhatever-1"
              role="listbox"
              square={true}
              style={Object {}}
            >
              <div
                className="MuiPaper-root-18 MuiPaper-elevation2-22"
                id="react-autowhatever-1"
                role="listbox"
                style={Object {}}
              />
            </Paper>
          </WithStyles(Paper)>
        </div>
      </Autowhatever>
    </Autosuggest>
  </div>
</SymbolSearch>
`;
